flasgger/base.py

```diff
@@ -351,7 +359,11 @@ def get_apispecs(self, endpoint='apispec_1'):
         for rule, verbs in specs:
             operations = dict()
             for verb, swag in verbs:
-                definitions.update(swag.get('definitions', {}))
+                update_dict = swag.get('definitions', {})
+                if type(update_dict) == list and type(update_dict[0]) == dict:
+                    # pop, assert single element
+                    update_dict, = update_dict
```

Comment:
not sure if this line works on all supported python versions, isn't best to use `update_dict[0]` directly?

Reply:
I'm pretty sure tuple unpacking is pretty standard Python, even going back to 2 https://docs.python.org/2/tutorial/datastructures.html#tuples-and-sequences